src/marqo/tensor_search/models/preprocessors_model.py

```diff
@@ -1,7 +1,7 @@
 from typing import Optional, Any
 
 from marqo.base_model import MarqoBaseModel
-from marqo.s2_inference.multimodal_model_load import Modality
+from marqo.core.inference.modality_utils import Modality
```

Comment:
Shouldn't we be importing from the source? Which is:
```
marqo.core.inference.api
```
based on the import in `marqo.core.inference.modality_utils`